tests/test_everything_all_at_once.py

```diff
@@ -129,13 +129,13 @@ def check_pair_idx_C(self, dadb, idx):
         da, db = dadb
         self.assertEqual(integral_category(*idx), "C")
         i, j, k, l = idx
-        exc = Excitation.exc_degree(da, db)
+        exc = da.exc_degree(db)
         self.assertIn(exc, ((1, 0), (0, 1)))
         if exc == (1, 0):
             (dsa, _), (dsb, _) = da, db
         elif exc == (0, 1):
             (_, dsa), (_, dsb) = da, db
-        h, p = PhaseIdx.single_exc_no_phase(dsa, dsb)
+        h, p = da.single_exc_no_phase(dsa, dsb)
```

Comment:
It feel werid as it's not related to `da`, but whatever :)

Reply:
Yeah agreed. I switched to

`Determinant.signle_exc_no_phase(dsa, dsb)`

It is a static method of the Determinant class

Reply:
This is also solely aa function used for testing